lib/gpu/analysis_options.yaml

```diff
@@ -1,7 +0,0 @@
-# Copyright 2013 The Flutter Authors. All rights reserved.
-# Use of this source code is governed by a BSD-style license that can be
-# found in the LICENSE file.
-
-linter:
-  rules:
-    - public_member_api_docs
```

Comment:
For the Flutter GPU analyzer failures, would it make sense to keep this rules file until I'm able to migrate it to the new rules we've agreed on for Flutter Engine Dart files?